components/menu/calendar/Screens/DashboardScreen/index.js

```diff
@@ -198,6 +200,36 @@ export default class DashboardScreen extends Component {
        this.props.navigation.navigate('FetchScreen');
      }
 
+     /**
+     * @param {String} description - A location to go to
+     * Navigates to the home component
+     */
+     sendDirections = (description) => {
+       if (!description) {
+         alert('There is no address or description for this event.');
```

Comment:
Perhaps this could be an improvement, but an alert like this will not be translated appropriately if not done in localization-context.js. See example **LOCATION_ALERT_MESSAGE:**